test/parallel/test-runner-module-mocking.js

```diff
@@ -514,41 +514,21 @@ test('CJS mocks can be used by both module systems', async (t) => {
   const cjsMock = t.mock.module(cjsFixture, {
     namedExports: { fn() { return 42; } },
   });
-  let esmImpl = await import(cjsFixture);
+  let esmImpl = await import(pathToFileURL(cjsFixture));
   let cjsImpl = require(cjsFixture);
 
   assert.strictEqual(esmImpl.fn(), 42);
   assert.strictEqual(cjsImpl.fn(), 42);
 
   cjsMock.restore();
 
-  esmImpl = await import(cjsFixture);
+  esmImpl = await import(pathToFileURL(cjsFixture));
   cjsImpl = require(cjsFixture);
 
   assert.strictEqual(esmImpl.default.string, 'original cjs string');
   assert.strictEqual(cjsImpl.string, 'original cjs string');
 });
 
-test('ESM mocks can be used by both module systems', async (t) => {
```

Comment:
Why remove this test? (Other changes LGTM)

Reply:
I couldn't find a way to refactor it. Either we use an absolute path for `require` and a file: URL for `import()` (but that's already the previous test), either we use the same relative specifier (but that's already the following test); so it seemed to me it was testing a "real" use case, or rather it's testing some particular case where an absolute path coincides with an origin-relative URL – but we cannot guarantee that, it depends on where the node repo was cloned.